CONTRIBUTING.md

```diff
@@ -33,22 +33,46 @@ To send us a pull request, please:
 2. Modify the source; please focus on the specific change you are contributing. If you also reformat all the code, it will be hard for us to focus on your change.
 3. Include unit tests when you contribute new features or make bug fixes, as they help to a) prove that your code works correctly, and b) guard against future breaking changes to lower the maintenance cost.
 4. Ensure local tests pass.
-5. Update the [changelog](https://github.com/aws/sagemaker-python-sdk/blob/master/CHANGELOG.rst) with a description of your changes for any code contribution.
-6. Commit to your fork using [clear and imperative commit messages](https://chris.beams.io/posts/git-commit/).
-7. Send us a pull request, answering any default questions in the pull request interface.
-8. Pay attention to any automated CI failures reported in the pull request, and stay involved in the conversation.
+5. Use commit messages (and PR titles) that follow [these guidelines](#commit-message-guidelines).
+6. Send us a pull request, answering any default questions in the pull request interface.
+7. Pay attention to any automated CI failures reported in the pull request, and stay involved in the conversation.
```

Comment:
doesn't have to be this PR, but maybe we should use `#` for numbering so that we don't have to maintain the actual numbers